packages/mdc-top-app-bar/mdc-top-app-bar.scss

```diff
@@ -145,5 +145,13 @@
   }
 }
 
+.mdc-top-app-bar--fixed {
+  position: fixed;
```

Comment:
We probably want some kind of `transition` for `box-shadow` to make the transition between `--fixed` and `--fixed-scrolled` a little smoother.